app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -1,17 +1,19 @@
-<% unless sender == recipient %>
-  <% follow_request = current_user.sent_follow_requests.find_by(recipient: recipient) %>
+<div class="<%=dom_id(recipient)%>_buns">
```

Comment:
I noticed that you did not include the sender id in your class, but you passed in a sender id in your create.js.erb and destroy.js.erb. Without the the dom_id for sender, your code will only work for one person and nobody else. Could you add the dom_id for the sender here and update your create.js.erb as well as your destroy.js.erb?

Reply:
changed!